src/test/system_tests/ceph_s3_tests/s3-tests-lists/s3_tests_pending_list.txt

```diff
@@ -27,14 +26,13 @@ s3tests_boto3.functional.test_s3.test_lifecycle_expiration_versioning_enabled
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_tags2
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_versioned_tags2
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_noncur_tags1
-s3tests_boto3.functional.test_s3.test_lifecycle_set_date
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_header_put
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_header_head
 s3tests_boto3.functional.test_s3.test_lifecycle_expiration_header_tags_head
+s3tests_boto3.functional.test_s3.test_lifecycle_transition_set_invalid_date
```

Comment:
I don't understand why we moved from passed to failed (we try to decrease the number of tests in the pending list). If something in the test itself is should be fixed, please open an issue/PR in the [Ceph](https://github.com/ceph/s3-tests) repository with all the details and update this file: [ceph_s3_tests_pending_list_status.md](https://github.com/noobaa/noobaa-core/blob/master/docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md)

Reply:
As I understand it passed because we sent an error in case the rule was with a prefix which was fixed here, but we don't check the date is valid as needed in the test.
@nadavMiz, please write us if you want to add a commit with this change or a separate PR.

Reply:
Would you please update the file [ceph_s3_tests_pending_list_status.md](https://github.com/noobaa/noobaa-core/blob/master/docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md) and add an explanation (you can also add a link to these comments)?

Reply:
added it to the commit